build.gradle.kts

```diff
@@ -106,3 +106,7 @@ tasks.test {
   finalizedBy(tasks.jacocoTestReport)
   useJUnitPlatform()
 }
+
+tasks.checkstyleMain {
```

Comment:
To review: this is not a great solution, but not sure if having a different version of the plugin for this component is better.